### DISCUSSION TITLE
Custom domain

### DISCUSSION THREAD

**Discussion by bzoum** [plausible/analytics#1092] [Anything Else]
Hello,
Do the custom domains still work? I just realized that I don't receive any analytics since 10 May? I wanted to verify the settings and I only can remove.
Why?

**Comment by metmarkosaric** [comment]
Hello! Existing custom domains still work all fine without problem but they are deprecated. We recommend to all those who use the custom domain and all the new users to proxy the script instead https://plausible.io/docs/proxy/introduction

**Reply by bzoum** [reply]
Hello.
I checked your doc and I cannot follow your tuto.
I have cloudflare but I already use a worker and we cannot have two workers.
I don't have next or netlify (I even don't know what they do...).

So I'm no other way than finding another analytics tool?

**Reply by metmarkosaric** [reply]
You can have multiple workers on the free Cloudflare account. You can also use our default script or your [custom domain](https://plausible.io/docs/custom-domain/). All of these options work. Proxy is an optional brand new feature and we also plan to add proxy instructions for even more platforms in the future for those who don't use Cloudflare/Netlify/Vercel but want to use the proxy.
